### DISCUSSION TITLE
Record all Commands in Wazah?

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#7518] [Unsupported Versions] [answered]
Hi All,

I'm using the follow audit.rules as a base: https://github.com/Neo23x0/auditd

How do I record all commands run by any user using Wazuh? At the moment it's only recording sudo commands.

**Comment by dougburks** [comment, marked as answer]
Looking at those audit.rules, it looks like they only log commands executed by root.  From https://github.com/Neo23x0/auditd/blob/master/audit.rules#L451-L453:
```
## Root command executions
-a always,exit -F arch=b64 -F euid=0 -S execve -k rootcmd
-a always,exit -F arch=b32 -F euid=0 -S execve -k rootcmd
```
I haven't tried it, but you might try removing `-F euid=0` and see if that logs for any user.
